pyensembl/gtf.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, gtf_path, cache_directory_path=None):
 
     def __eq__(self, other):
         return (
-            isinstance(other, GTF) and
+            (other.__class__ == GTF) and
```

Comment:
not important, but just noting I think this change means you can't subclass GTF and have equality work. Not sure if that's intentional